akka-remote/src/main/protobuf/ContainerFormats.proto

```diff
@@ -41,6 +41,10 @@ message ActorRef {
   required string path = 1;
 }
 
+message Some {
+  optional Payload value = 1;
```

Comment:
I think it would be better to rename it to `Option` and not use different manifests for None and Some